### ISSUE TITLE
Bug: failed to parse timezone

### ISSUE THREAD

**Issue by weixcloud** [caas-team/GoKubeDownscaler#61]
## Issue

I deployed the operator in my local cluster for testing, When I use the timespan with any time zone like Australia/Sydney, the operator printed the following error:

```
annotation: failed to parse relative timespan: failed to parse timezone: 
unknown time zone Australia/Sydney" workload=demo-deploy
 namespace=xxx
```

### Proposal

There are two options to solve it:
* `add _ "time/tzdata"` in the main.go
* Or change the docker image by adding the following line:
```
FROM scratch
COPY --from=builder /usr/local/go/lib/time/zoneinfo.zip /zoneinfo.zip
ENV ZONEINFO=/zoneinfo.zip
```

**Comment by weixcloud** [comment]
@jonathan-mayer Could you please have a look at it? It should be easy to fix with the option 1. Thanks

**Comment by jonathan-mayer** [comment]
Hey @weixcloud, sorry for the delayed response, I am currently on vacation. Thanks for finding this issue. I will have a look at this and release a new version of the downscaler once its fixed.

**Comment by jonathan-mayer** [comment]
I've implemented a fix in #62 it should be merged tomorrow after being reviewed and a new version should automatically be released.
